src/main.js

```diff
@@ -118,6 +118,12 @@ function saveCurrentCover() {
   </div>`
   }
 }
+
+function deleteMiniCover(clickLocation) {
+    var miniCover = clickLocation.target.closest("section > div")
+    miniCover.remove()
+  }
+
```

Comment:
This function is called when the user double-clicks the saved cover section. It uses the click location to find the closest mini-cover div within that section, then removes the entire div.